### DISCUSSION TITLE
GKE deploy error

### DISCUSSION THREAD

**Discussion by josex2r** [emqx/emqx-operator#617] [Q&A] [answered]
Hi.
I'm trying to deploy an EMQX cluster on GKE but I'm totally stuck u.u

Running everything in my local cluster seems to work fine, but when I try to deploy the same as I ran in my local machine in GKE I'm not able to deploy EMQX.

Steps:
1. Install `cert-manager` in GKE. It works as expected
2. Install `emqx-operator` in GKE. It works, the service and the webhook are running. I'm also able to call it from another container and everything seems to work.
![Screenshot 2023-02-14 at 15 24 01](https://user-images.githubusercontent.com/3719730/218773924-98bf15be-0465-4d9a-aaa4-25b03869eda3.png)
3. Deploy EMQX fail, timeout error.
![Screenshot 2023-02-14 at 15 55 59](https://user-images.githubusercontent.com/3719730/218774525-2a05b20c-2278-49d0-b97f-f7c07ab44283.png)

Someone has been able to deploy it on google Cloud?
Thank you.

**Comment by Rory-Z** [comment]
Hello @josex2r Please make sure the endpoint of the `emqx-operator-webhook-service` is not empty, and retry Steps 3
```
$ kubectl get endpoints -n emqx-operator-system -o json emqx-operator-webhook-service -o json | jq '.subsets[].addresses'
[
  {
    "ip": "172.17.0.3",
    "nodeName": "minikube",
    "targetRef": {
      "kind": "Pod",
      "name": "emqx-operator-controller-manager-77c68dc5bc-wjmd9",
      "namespace": "emqx-operator-system",
      "resourceVersion": "1851420",
      "uid": "a69321cb-b2f3-43b7-af04-34f33a81d32b"
    }
  }
]
```

**Comment by josex2r** [comment, marked as answer]
Thank you @Rory-Z 

Finally, I've been able to deploy on GCP 👏  and I also opened a pull-request to improve the docs (#620)
